src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -99,6 +105,27 @@
   private static final String dropScopedCredentialTemplate =
       "DROP DATABASE SCOPED CREDENTIAL [<resourceName>];";
 
+  public List<String> getRefIds(String tableName, SynapseColumn refColumn) {
```

Comment:
Aren't refIds UUIDs?  Would it make sense to make this return a list of UUIDs?

Reply:
It seemed a smaller change to make these strings to match existing GCP code rather than change all the existing GCP code to UUID. I could totes do it though!

Reply:
https://broadworkbench.atlassian.net/browse/DR-2088 follow on ticket created.